src/components/Artists/ArtistItem.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ArtistItem = ({ artistName }) => (
+  <p>{artistName}</p>
```

Comment:
I could totally be wrong, but it seems like this is an unnecessary component, unless you planned on expanding it later